pkg/util/log/buffered_sink.go

```diff
@@ -371,6 +381,15 @@ func (bs *bufferedSink) runFlusher(stopC <-chan struct{}) {
 	}
 }
 
+// maybeLogFlushError will log err to the OPS channel with the ERROR severity
+// ONLY if the bufferedSink has flushed successfully before. This is to prevent
+// creating a cycle of noise when a sink is intentionally unavailable.
+func (bs *bufferedSink) maybeLogFlushError(err error){
+	if bs.hasFlushedSuccessfully.Load() {
+		Ops.Errorf(context.Background(), "logging error from %T: %v", bs.child, err)
+	}
+}
```

Comment:
Consider using `log.EveryN` instead to only log, say, every few minutes if this is happening. This way we can capture connection errors on startup and intermittently. Plus, if the connection errors arise after the sink is up and running, they still won't spam logs.

see: https://github.com/cockroachdb/cockroach/blob/c097a16427f65e9070991f062716d222ea5903fe/pkg/util/log/every_n.go